front/src/components/student-registration/student-registration-page/student-registration-page.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import { hideStudentRegisteredModal } from '../../../common/redux/actions';
+import { connect } from 'react-redux';
+import { Modal, Row, Col } from 'antd';
+import StudentRegistrationForm from '../student-registration-form';
+
+class StudentRegistrationPage extends Component {
+  constructor(props) {
+    super(props);
+    this.handleHideModal = this.handleHideModal.bind(this);
+  }
+
+  handleHideModal() {
+    this.props.dispatch(hideStudentRegisteredModal());
+  }
+
+  render() {
+    const { errorMessage, isRegistered, isModalVisible, match, dispatch } = this.props;
+
+    if (!isRegistered) {
+      return (
+        <>
+          <StudentRegistrationForm
+            token={match.params.token}
+            dispatch={dispatch}
+          />
+          <Modal
```

Comment:
I don't think that it should be an actual modal. It is better to  put register form inside the card and that's all

Reply:
Removed modal, and added Notification instead